src/feat/feature-fbank.h

```diff
@@ -71,6 +73,8 @@ struct FbankOptions {
                    "to change other parameters).");
     opts->Register("use-log-fbank", &use_log_fbank,
                    "If true, produce log-filterbank, else produce linear.");
+    opts->Register("use-power", &use_power,
+		   "If true, use power in filterbank analysis, else magnitude.");
```

Comment:
please check that this line doesn't  exceed 80 characters.  And replace tabs with spaces...